helm-chart/binderhub/files/binderhub_config.py

```diff
@@ -46,10 +46,11 @@ def get_value(key, default=None):
 for section, sub_cfg in get_value("config", {}).items():
     c[section].update(sub_cfg)
 
-if get_value("dind.enabled", False) and get_value("dind.hostSocketDir"):
-    c.BinderHub.build_docker_host = "unix://{}/docker.sock".format(
-        get_value("dind.hostSocketDir")
-    )
+containerBuilderPod = get_value("containerBuilderPod")
```

Comment:
It took me a while to figure this out.... although `get_value()` reads from `values.yaml`
https://github.com/jupyterhub/binderhub/blob/bef386a4875f511368e38b6af1ca98f233eb6fa8/helm-chart/binderhub/files/binderhub_config.py#L15
it confusing doesn't read the full `values.yaml`, instead it's only a subset of keys:
https://github.com/jupyterhub/binderhub/blob/bef386a4875f511368e38b6af1ca98f233eb6fa8/helm-chart/binderhub/templates/secret.yaml#L18-L19
so `secret.yaml` needs to be
```yaml
  values.yaml: |
    {{- pick .Values "config" "containerBuilderPod" "cors" "dind" "extraConfig" "pink" | toYaml | nindent 4 }}
```

Reply:
I have completely missed that and it just hit me on Friday. It looks like I managed to test drive BinderHub with only images that were already present and did not require a fresh pull.

Thanks for the fix !